src/epub/text/the-inferiority-complex-of-old-sippy.xhtml

```diff
@@ -0,0 +1,293 @@
+<?xml version="1.0" encoding="utf-8"?>
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" epub:prefix="z3998: http://www.daisy.org/z3998/2012/vocab/structure/, se: https://standardebooks.org/vocab/1.0" xml:lang="en-GB">
+	<head>
+		<title>The Inferiority Complex of Old Sippy</title>
+		<link href="../css/core.css" rel="stylesheet" type="text/css"/>
+		<link href="../css/local.css" rel="stylesheet" type="text/css"/>
+	</head>
+	<body epub:type="bodymatter z3998:fiction">
+		<article id="the-inferiority-complex-of-old-sippy" epub:type="volume se:short-story">
```

Comment:
`volume` is no longer used for short story semantics, you can remove it from both

Reply:
It appeared in 5 other stories so I removed them as well